app/src/main/java/org/oppia/android/app/testing/ConceptCardFragmentTestActivity.kt

```diff
@@ -31,15 +31,15 @@ class ConceptCardFragmentTestActivity : InjectableAppCompatActivity(), ConceptCa
   }
 
   private fun getConceptCardFragment(): ConceptCardFragment? {
-    return supportFragmentManager.findFragmentByTag(TAG_CONCEPT_CARD_DIALOG) as ConceptCardFragment?
+    return supportFragmentManager.fragments.singleOrNull { fragment ->
+      fragment is ConceptCardFragment
+    } as ConceptCardFragment?
```

Comment:
```suggestion
    return supportFragmentManager.fragments.filterIsInstance<ConceptCardFragment>().single()
```

Bit more robust and equivalent.

Reply:
Done, but kept singleOrNull() instead of single() to keep the original method signature.

Reply:
Ah, I think I meant ``singleOrNull`` originally (per your existing code)--good catch.